### ISSUE TITLE
The paper-dropdown-menu is not accessible when using ChromeVox or VoiceOver

### ISSUE THREAD

**Issue by keshshvets** [PolymerElements/paper-dropdown-menu#295]
### Description
The `paper-dropdown-menu` component is not accessible when using ChromeVox or VoiceOver. It is being announced as a text field (which also happens to be non-editable) instead of a `combobox`/`button` role.

Note that the light element mostly works as expected (it's reported as a combo box), except that you can't type into the element, as would be implied by the `combobox` role.

### Expected outcome
After navigating to a `paper-dropdown-menu`, the screen reader should announce that the component is a `combobox` or a `button` (with popup content).

### Actual outcome
After navigating to a `paper-dropdown-menu`, the screen reader announces that the component is a text field. Trying to type into the field does not work, leading to further confusion.

### Live Demo
Use the existing demo as a reference: https://www.webcomponents.org/element/PolymerElements/paper-dropdown-menu/demo/demo/index.html

### Steps to reproduce
Visit the live demo, enable the screen reader, and navigate to the `paper-dropdown-menu` component. Listen to what the screen reader announces.

### Browsers Affected
- [X] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

**Comment by bicknellr** [comment]
This should be fixed by #317 and available in v3.1.0.
